apps/browser/src/platform/browser/browser-api.ts

```diff
@@ -17,11 +17,35 @@ export class BrowserApi {
     return chrome.runtime.getManifest().manifest_version;
   }
 
-  static getWindow(windowId?: number): Promise<chrome.windows.Window> | void {
+  /**
+   * Gets the current window or the window with the given id.
+   * @param {number} windowId
+   * @returns {Promise<chrome.windows.Window>}
+   */
+  static async getWindow(windowId?: number): Promise<chrome.windows.Window> {
     if (!windowId) {
-      return;
+      return BrowserApi.getCurrentWindow();
     }
 
+    return await BrowserApi.getWindowById(windowId);
+  }
+
+  /**
+   * Gets the currently active browser window
+   * @returns {Promise<chrome.windows.Window>}
+   */
+  static async getCurrentWindow(): Promise<chrome.windows.Window> {
+    return new Promise((resolve) =>
+      chrome.windows.getCurrent({ populate: true }, (window) => resolve(window))
```

Comment:
We could use the shorthand here.
```suggestion
      chrome.windows.getCurrent({ populate: true }, resolve)
```

Reply:
Good call, done.